web/webkit/src/main/scala/net/liftweb/http/HtmlNormalizer.scala

```diff
@@ -0,0 +1,291 @@
+package net.liftweb
+package http
```

Comment:
The package might be better named `web`, but I think it's extremely nice from a framework usability perspective that you can import `net.liftweb.http._` and be good to go on all non-AJAX stuff (and some AJAX stuff as well).